.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
i would actually strongly discourage husky; git hooks included in the repo are a security concern, which is why you can't do this directly in git, and in the past i've seen husky inside node_modules install git hooks in a corporate internal repo.

Reply:
Noted!

> linting should be done in CI, and isn't important to do anywhere else - and anything not enforced in CI isn't enforced at all, since local git hooks can always be bypassed.

Yes, ideally we'd have this setup in our CI which we still need to setup. I can remove husky in favor of `.githooks/pre-push`

Reply:
To be clear, I'm suggesting avoiding any git hooks in the repo whatsoever, as they are always a security risk - it's just that husky is particularly dangerous in my experience.

Reply:
@ljharb Got it. Thanks for the context. I'm going to limit this PR to formatting & linting of current files

Reply:
For setting up CI, i'd be happy to pair with you on setting up github actions, which is already enabled for this org.

Reply:
that'd be great! will reach out to setup some time